packages/@ourworldindata/core-table/src/Transforms.ts

```diff
@@ -217,88 +271,122 @@ enum WhereOperators {
 }
 // Todo: add tests/expand capabilities/remove?
 // Currently this just supports `columnSlug where someColumnSlug (isNot|is) this or that or this`
-const where = (
-    columnStore: CoreColumnStore,
-    columnSlug: ColumnSlug,
-    conditionSlug: ColumnSlug,
-    ...condition: string[]
-): CoreValueType[] => {
-    const values = columnStore[columnSlug]
-    const conditionValues = columnStore[conditionSlug]
-    const operator = condition.shift()
-    let passes: (value: any) => boolean = () => true
-    if (operator === WhereOperators.isNot || operator === WhereOperators.is) {
-        const result = operator === "isNot" ? false : true
-        const list = condition.join(" ").split(" or ")
-        const set = new Set(list)
-        passes = (value: any): boolean => (set.has(value) ? result : !result)
-    } else if (operator === WhereOperators.isGreaterThan)
-        passes = (value: any): boolean => value > parseFloat(condition.join(""))
-    else if (operator === WhereOperators.isGreaterThanOrEqual)
-        passes = (value: any): boolean =>
-            value >= parseFloat(condition.join(""))
-    else if (operator === WhereOperators.isLessThan)
-        passes = (value: any): boolean => value < parseFloat(condition.join(""))
-    else if (operator === WhereOperators.isLessThanOrEqual)
-        passes = (value: any): boolean =>
-            value <= parseFloat(condition.join(""))
-
-    return values.map((value, index) =>
-        passes(conditionValues[index]) ? value : ErrorValueTypes.FilteredValue
-    )
+const where: Transform = {
+    params: [
+        { type: TransformParamType.DataSlug },
+        {
+            type: TransformParamType.ColumnSlug,
+        },
```

Comment:
Open question to me right now: What's the difference between the two?

Reply:
`ColumnSlug` is a slug for a column with entities, years or data values. `DataSlug` only refers to columns with data values. So, every `DataSlug` is a `ColumnSlug` but not every `ColumnSlug` is a `DataSlug`